src/components/filterbox.pcss

```diff
@@ -2,12 +2,26 @@
   background: #fff;
   box-sizing: border-box;
   color: #333;
-  font-size: 1.5rem;
+  font-size: 1rem;
   height: 100%;
   outline: none;
   position: relative;
   user-select: none;
 
+  *::-webkit-scrollbar {
```

Comment:
Not too fond of us fiddling with scrollbar styles (which are affected by e.g. theming/accessibility settings in their OS).

Reply:
Sure. I'll remove this.